src/mcts.rs

```diff
@@ -93,23 +98,61 @@ impl<'a> Searcher<'a> {
 
             nodes += 1;
 
-            if nodes % 128 == 0 {
+            if nodes & 255 == 0 {
                 if self.abort.load(Ordering::Relaxed) {
```

Comment:
Same goes for the other instances of this, please just do `nodes % 256`, it conveys the intent clearer and the optimisation is done even in debug mode.